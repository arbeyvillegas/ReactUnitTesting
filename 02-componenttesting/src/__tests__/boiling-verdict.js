import React from 'react';
import { shallow } from 'enzyme';
import BoilingVerdict from '../boiling-verdict';
import toJson from 'enzyme-to-json';


it("when celsius is 50 the water would not boil", () => {
    const wrapper = shallow(<BoilingVerdict celsius={50}/>);

    expect(toJson(wrapper)).toMatchSnapshot();
});

it("when celsius is 120 the water would boil", () => {
    const wrapper = shallow(<BoilingVerdict celsius={120}/>);

    expect(toJson(wrapper)).toMatchSnapshot();
});