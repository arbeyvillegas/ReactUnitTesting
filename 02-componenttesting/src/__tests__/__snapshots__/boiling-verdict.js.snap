// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`when celsius is 50 the water would not boil 1`] = `
<p>
  The water would not boil.
</p>
`;

exports[`when celsius is 120 the water would boil 1`] = `
<p>
  The water would boil.
</p>
`;
